lib/OpenQA/FakeApp.pm

```diff
@@ -43,24 +47,53 @@ has schema => sub { OpenQA::Schema::connect_db() };
 
 sub setup_log {
     my ($self) = @_;
-    if ($self->log_dir) {
-        # So each worker from each host get it's own log (as the folder can be shared). Hopefully the machine hostname
-        # is already sanitized. Otherwise we need to check
-        my $logfile
-          = catfile($self->log_dir, hostname() . (defined $self->instance ? "-${\$self->instance}" : '') . ".log");
+    my ($logfile, $logdir, $level, $log) = ('', '', '', undef, '');
```

Comment:
i think this is not necessary

Reply:
Those are the default values.
For example for the logdir variable. If the object isn't of type FakeApp, then the object doesn't have a logdir function. Since i'm setting that value in the if:

>    if ($self->isa('OpenQA::FakeApp')) {
>        $logdir = $self->log_dir;
>        $level  = $self->level;
>    }
>    else {
>        $log = $self->log;
>        OpenQA::FakeApp::read_config($self);
>        $level = $self->config->{logging}->{level} || 'debug';
>    }

then i can use it later like:

>    if ($logfile && $logdir) {
>        my $logfile = catfile($logdir, $logfile);
>        $log = Mojo::Log->new(handle => \*STDOUT, level => $self->level);
>    }

Or did i understood wrongly your comment?

Reply:
It's ok to use different variables - but defaults values looks really weird :)
Why can't just be: 
```perl
my ($logfile, $logdir, $level, $log);
```

Reply:
Got your point :-)